frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.tsx

```diff
@@ -339,7 +339,10 @@ export function ObjectDetailView({
     });
   }, [actionId, zoomedRowID]);
 
-  const initialValues = useMemo(() => ({ id: zoomedRowID }), [zoomedRowID]);
+  const initialValues = useMemo(
+    () => ({ id: zoomedRowID || null }),
```

Comment:
Technically this would convert `0` to `null`, maybe it should be `ID ?? null`?

Reply:
95fdaad44f89de246dfb4de31a9fcdcb5b8381a6